source/tutorials/filtering.rst

```diff
@@ -129,6 +138,15 @@ For example, filtering the table to contain only samples from subject 1 is perfo
       --p-where "Subject='subject-1' AND BodySite='gut'" \
       --o-filtered-table subject-1-gut-filtered-table.qza
 
+The ``OR`` keyword syntax is similar to the ``AND`` keyword syntax, but specifies that either of the expressions can be true for a sample to be retained. For lack of a more relevant application to the example data being used here, the ``OR`` keyword in this example is applied to retain samples all of the samples where ``BodySite`` is ``gut`` *or* ``ReportedAntibioticUsage`` is ``Yes`` in ``sample-metadata.tsv``.
```

Comment:
@ebolyen, I expanded this (and the ``AND`` discussion). I agree that a truth table (or something along those lines) would be useful, but I'd kind of rather point to a discussion of these operators somewhere else (e.g., an online CS resource). Or, I'm just getting burned out on docs for this release. Let me know if you think that this is good as-is (with my latest changes), needs additional content right now, or needs additional content which we can create an issue for.